src/ContinuumAbsorption/absorption_ff_neutral_metals_molecules.jl

```diff
@@ -0,0 +1,422 @@
+"""
+function _combined_john_neg_ion_absorption(ν, T, ndens_neutral, ne, short_wavelength_interp,
+                                           long_wavelength_interp)
+    # this involves a small amount of guessing, but we will confirm this is correct by comparing
+    # He⁻ absorption coefficient with our reference
+
+    λ_cm = c_cgs / ν # in cm
+    Pₑ = ne * kboltz_cgs * T # partial pressure contributed by electrons
+
+    cross_section = if λ_cm > 1e-3
+        (λ_cm * λ_cm * 1e16) * long_wavelength_interp(T)
+    else
+        short_wavelength_interp(λ_cm, T)
+    end
+
+    ndens_neutral * Pₑ * cross_section
+end
+
+
+# load free-free absorption data from Table I of for interactions named after negative ions (but
+# involve neutral species) John T. L. 1975, MNRAS, 170, 5. This data is for large λ (>~ 10 μm).
+#
+# MARCS/Turbospectrum only uses this data to compute for a small subset of species
+const _long_wavelength_ff_interps = let
+    T_vals = SA[ 50.0,    100.0,    500.0,    1000.0,   5000.0,   10000.0,   25000.0]
+
+    table = Dict(
+        "He"  => [    1.34,     0.968,    0.475,     0.353,    0.173,     0.13,      0.07  ],
+        "Ne"  => [    0.107,    0.0823,   0.0604,    0.056,    0.041,     0.0328],  #-----
+        "Ar"  => [    1.4,      0.673,    0.057,     0.019,    0.038,     0.062,     0.097 ],
+        "Kr"  => [    6.17,     3.37,     0.527,     0.17,     0.034,     0.073,     0.138 ],
+        "Xe"  => [   27.7,     13.8,      1.74,      0.548,    0.127,     0.234,     0.303 ],
+        "Li"  => [   20.5,     41.6,     60.0,      28.0,      0.93,      0.062,     0.012 ],
+        "Na"  => [   18.6,     24.6,     47.0,      26.0,      2.7,       1.3],     #-----
+        "Cs"  => [ 1695.0,    853.0,     88.0,      57.0,      4.7,       2.8,       1.2   ],
+        "Hg"  => [    5.09,     3.02,     0.845,     1.54],   #-----      -----      -----
+        "N"   => [    0.263,    0.195,    0.13,      0.12,     0.1,       0.0904,    0.0699],
+        "O"   => [    0.280,    0.202,    0.119,     0.103,    0.0743,    0.0654,    0.0512],
+        "H2"  => [    1.9,      1.47,     0.866,     0.703,    0.429,     0.29,      0.112 ],
+        "N2"  => [    0.597,    0.554,    0.506,     0.459,    0.333,     0.293,     0.14  ],
+        "O2"  => [    0.762,    0.54,     0.343,     0.261,    0.171,     0.114,     0.076 ],
+        "CO"  => [   21.3,      1.14,     0.642,     0.605,    0.45,      0.28,      0.15  ],
+        "CO2" => [   43.5,     21.7,      4.1,       1.75,     0.192,     0.16,      0.13  ],
+        "H20" => [  996.0,    342.0,     28.0]      #-----     -----      -----      -----
+    )
+
+    Dict(k => LinearInterpolation(view(T_vals, 1:length(v)), v.*1e-34, extrapolation_bc = Throw())
+         for (k,v) in table)
+end
+
+
+# load free-free absorption data from John 1975, MNRAS, 172, 305. This data is valid for
+# 0.5 μm ≤ λ ≤ 10 μm. Most of these tables share a common set of λ and T values.
```

Comment:
```suggestion
# Most John1975b tables share a common set of λ and T values.
```

The comment starting with "loads..." was confusing to me when I was trying to understand the structure of this file.